soccer/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ set(ROBOCUP_LIB_SRC
     "vision/util/VisionFilterConfig.cpp"
     "vision/VisionFilter.cpp"
     "WindowEvaluator.cpp"
-)
+        Context.hpp)
```

Comment:
Tabbing feels off here

Reply:
Uh yeah...actually that shouldn't have even been added.

Reply:
I think CLion did it automatically. I'll get rid of it.